apps/web/vercel.json

```diff
@@ -18,7 +18,7 @@
       "memory": 1024
     },
     "pages/api/trpc/slots/[trpc].ts": {
-      "memory": 1024
+      "memory": 3008
```

Comment:
Wouldn't this also increase the memory for the getSchedule as well?

Reply:
No. Current getSchedule is under public.slots.getSchedule

Reply:
Configured here. https://github.com/calcom/cal.com/pull/17277/files#diff-361f945cebcf11354b7400e3c2b5f31c66b73f5c3a5b3f428a0a0f78d3bc898bR17

Reply:
I see. Curious on why we also [have it here](https://github.com/calcom/cal.com/pull/17277/files#diff-a36eefff6298ee93f3722ad3e7bd646dd45f73eba1b1a805a6577ed86c1792ceR20).

Reply:
Yeah I guess it was originally built there and then someone decided that since it's public, it should also live on the public router?

Reply:
Does make me wonder though - should we put this in a separate router altogether. Because this would mean that reserveSlot and removeSelectedSlotMark would now use 3x the memory/vCPU/cost basically for no reason. Perhaps a separate "high-perf" handler?

Reply:
From a practical design perspective, I don't like this at all, but we are limited based on how Vercel allows us to scale these alongside tRPC.